Azure/azure-rest-api-specs
# New API Version 2017-08-01: Adding Sku property into publicIP and loa…

…d balancer (#1528)

* Updating documentation for Microsoft.VisualStudio with project resource APIs. Corrections to existing examples. (#1498)

* Added Api-Version 2017-07-01 for RecoveryServices.Backup Jobs (#1502)

* Added Api-Version 2017-07-01 for RecoveryServices.Backup Jobs

* Updated tag names

* Microsoft.Sql - Added data sync to package-2015-05-preview (#1510)

* Added advisors and data sync to package-2015-05-preview

* Removed advisors so current branch is only data sync

* Make MsDeploy and MsDeployLog Azure resources (#1519)

* Add public certificates, Functions Admin Token and MSDeploy APIs

* Revert resource definition changes and description change for site properties

* Make type and name readonly properties. Add ARM envelope to MSDeployStatus object

* Fix missing quotation issue.

* Add long running operation to MS deploy

* Make MsDeploy and MsDeployLog Azure resources

* Fix some AutoRest validation issues. Use dictionary for msdeploy.setparameters

* Ensure models are same across all schemas in Microsoft.Web

* Fix azure resource type across all models

* Remove conflicting ListOperations. It is not used for public Azure.

* no YAML in glob patterns for report scripts

* Copying 2017-06-01 to 2017-08-01

* Changes for new API version

* Updated script to run autorest with grouping swaggers based on config file (#1529)

* Removed the maxLength for entities (#1527)

* Update folder structure example. (#1489)

Agreed on the changes

* [Travis-ci] Oad in ci status (#1520)

* Breaking change build should fail if result contains error
Skipping newly added files
Incorporating review feedback

* testing

* log the result as oad.compare returns promise for caller

* Use published oad

* Revert "testing"

This reverts commit a9be94db024716e5a563c4bd418332be9aaf329b.

* Removed maxLength from entity name (#1531)

* This is to add Cognitive Services data-plane API specs starting with Face API. (#1467)

* Adding data plane spec starting with Face

* Move Spec to right folder

* Fixing validation issues.

* Fixing issues and adding a readme.md

* Fixing examples by removing decimals.

* Removing another decimal value

* Update readme.md

* Updating spec based on comments.

* Removing .gitignore.

* Relay: Removed the maxLength for entity names (#1535)

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
